frontend/src/lib/components/HabitsList.svelte

```diff
@@ -3,16 +3,38 @@
     export let time;
     export let iconCount;
     export let habitId;
- 
-    let menuOpen = false;
+    export let menuOpen;
+    export let storedId;
+   
+    function handleOpening(id){
```

Comment:
The function that handles the opening can be refactored. Let the page/component that is the actual habits list handle the closing of dropdowns. Maybe event forwarding might be a good idea in this case.

Not necessarily wrong, just would be a cleaner code